starcounter-include.html

```diff
@@ -246,6 +247,7 @@
          * @param  {Mixed} value New value
          */
         StarcounterIncludePrototype._notifyPath = function(path, value) {
+            var modelAlreadyLoaded = this.template.model == this.partial;
```

Comment:
1. I think there is a chance when `_notifyPath` is called before stamping (`this.template` assignment). Maybe `this.template && this.template.model == this.partial` would be more tolerant.
2. What about `null == null`? Maybe even 
```
this.template && this.template.model && this.template.model == this.partial
```

Reply:
I'd rename it to `sameModelAlreadyLoaded` as current name gave me lots of confusion in reasoning mentioned below.